monitor/ui/pom.xml

```diff
@@ -26,6 +27,9 @@
 						<goals>
 							<goal>compile</goal>
 						</goals>
+						<configuration>
+							<sourceLevel>1.7</sourceLevel> <!-- 1.6 or 1.7 -->
```

Comment:
Now I remember why this is here. It's not necessary for functionality but "the strange thing" is that it is not 1.8. Only 1.6. and 1.7. are allowed. That's why I left it there.